src/detect.c

```diff
@@ -828,12 +833,17 @@ static DetectRunScratchpad DetectRunSetup(
 
 #ifdef UNITTESTS
     p->alerts.cnt = 0;
+    p->alerts.discarded = 0;
+    p->alerts.suppressed = 0;
```

Comment:
Wasn't sure if these should stay here or go with the others (outside unittests definition)

Reply:
are they reset elsewhere as well?

Reply:
In `PACKET_RELEASE_REFS(p)`